### DISCUSSION TITLE
Request: Show frame count / duration of selected range

### DISCUSSION THREAD

**Discussion by gkertai** [videovillage/community#127] [Ideas]
When I mark an in/out range, it would be nice to see how many frames/seconds I have selected. Possibly inside the timecode pop-up.

**Comment by gregcotten** [comment]
I like this idea

**Comment by gkertai** [comment]
This could also make marker durations more useful (see videovillage/community#54) if there was a way to add a marker to the selected range.

E.g. I would hit I to mark in, play back a few frames, hit O to mark out, then hit ⇧M to create a marker for the selected range. The I/O range  could clear automatically (as per preference), or I might hit ⌥X to clear it manually.
